### PR TITLE
Created context and page design

### PR DESCRIPTION
**Pull request by IrisGalos**

### CODE REVIEW

File: src/views/main.tsx [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+
+  const { loggedIn } = context;
+
   return (
     <>
-      <MapAP />
+      {loggedIn ? (
+        <MapAP />
+      ) : (
+        <div className="main bg">
```
Comment (tomoiagahoratiu) [inline src/views/main.tsx:21]: Can we extract this code into a separate component? Maybe we will have to use it again somewhere else.

File: src/components/LogInForm.tsx [hunk 2]
```diff
@@ -1,30 +1,43 @@
...
   const [usersData, setUsersData] = useState<user[]>(users);
   const navigate = useNavigate();
 
+  const context = useContext(isLoggedIncontext);
+
+  if (!context) {
+    throw new Error("useContext must be used within an IsLoggedInProvider");
+  }
+
+  console.log(context);
```
Comment (tomoiagahoratiu) [inline src/components/LogInForm.tsx]: remove this console log

File: README.md [hunk 3]
```diff
@@ -65,4 +137,5 @@ The application is designed to provide **a straightforward and user-friendly int
 - [**Postman**](https://www.postman.com/)
 - [**Http methods**](https://www.w3schools.com/tags/ref_httpmethods.asp)
 - [**Http response codes**](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)
-- [**Create your own apis**](https://beeceptor.com/docs/sample-api-for-testing/)
\ No newline at end of file
+- [**Create your own apis**](https://beeceptor.com/docs/sample-api-for-testing/)
+>>>>>>> login
```
Comment (tomoiagahoratiu) [inline README.md]: delete this

File: README.md [hunk 4]
```diff
@@ -1,3 +1,75 @@
+<<<<<<< HEAD
```
Comment (tomoiagahoratiu) [inline README.md]: it seems you have some issues from rebasing

File: .gitignore [hunk 5]
```diff
@@ -18,16 +21,27 @@ dist-prod
...
+.env.production.local
+=======
 .vscode
 .DS_Store
+>>>>>>> login
 
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+<<<<<<< HEAD
```
Comment (tomoiagahoratiu) [inline .gitignore]: same rebase issues

File: README.md [hunk 6]
```diff
@@ -1,68 +1,70 @@
...
-- [**React Docs**](https://react.dev/learn) *(functional components and typescript)*
-- [**FlexBox game**](https://flexboxfroggy.com)
-- [**Postman**](https://www.postman.com/)
-- [**Http methods**](https://www.w3schools.com/tags/ref_httpmethods.asp)
-- [**Http response codes**](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)
-- [**Create your own apis**](https://beeceptor.com/docs/sample-api-for-testing/)
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (tomoiagahoratiu) [inline README.md:3]: please undo this changes, we want the old readme

**Comment by tomoiagahoratiu** [comment]
as per github you still have conflicts
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/cloudcargo2024/cloudcargo-web-application/pull/13/conflicts) or the  to resolve conflicts.
Conflicting files
README.md
src/App.tsx
src/components/LogInForm.tsx

you need to resolve them before merging

**Review by PopCristianGabriel — APPROVED** [review]
